tests/tests.h

```diff
@@ -187,6 +189,40 @@ namespace Testing
 
 
 
+// Get a uniformly distributed random value between min and max
+template<typename T=double>
+T random_value(const T &min, const T &max)
+{
+  return min+(max-min)*(Testing::rand()/static_cast<T>(RAND_MAX));
+}
+
+
+
+// Same as above, but allows one to leave 0 as a default minimum value,
+// and specify only the maximum
+template<typename T=double>
+T random_value(const T &max=1)
```

Comment:
indeed, as pointed by @jppelteret a single argument version is used in one test only, so please remove this funciton and instead add default to
```
random_value(const T &min=0., const T &max=1.)
```